plan/validator.go

```diff
@@ -285,6 +297,31 @@ func (v *validator) checkCreateTableGrammar(stmt *ast.CreateTableStmt) {
 	}
 }
 
+func (v *validator) checkCreateViewGrammar(stmt *ast.CreateViewStmt) {
+	if stmt.ViewName == nil {
```

Comment:
ViewName must not be nil, which is guaranteed by parser. This check is meaningless.